src/sql/azdata.proposed.d.ts

```diff
@@ -1790,6 +1795,73 @@ declare module 'azdata' {
 		 * The path of the parent node.
 		 */
 		parentNodePath: string;
+		/**
+		 * Filterable properties that this node supports
+		 */
+		filterableProperties?: NodeFilterProperty[];
+	}
+
+	export interface NodeFilterProperty {
+		/**
+		 * The name of the filter property displayed to the user
+		 */
```

Comment:
I think you also need another property: `supportedOperators`.

Reply:
nvm, on a second thought, I think it is not needed, you can tell the supported operators by the data type.